README.md

```diff
@@ -8,7 +8,6 @@ To install the library use JitPack:
 
 ```scala
 resolvers += "jitpack".at("https://jitpack.io")
-val repo = "com.github.VirtusLab.akka-safer-serializer"
 val commit = "master-SNAPSHOT" //name of a branch or commit
```

Comment:
Mention that for commit hash, no `-SNAPSHOT` is expected (at least that wouldn't make sense in principle 🤔 )